dev/bazel/dal.bzl

```diff
@@ -474,7 +474,12 @@ def _dal_module(name, lib_tag="dal", is_dpc=False, features=[],
         local_defines = local_defines + ([
             "DAAL_SYCL_INTERFACE",
             "ONEDAL_DATA_PARALLEL"
-        ] if is_dpc else []),
+        ] if is_dpc else []) + select({
+            "@config//:test_fp64_enabled": [],
```

Comment:
Looks a bit confusing: name of config is "test_fp64_enabled" but inside "ONEDAL_DISABLE_FP64_TESTS=1"

Reply:
See the next line. `select` with `//conditions:default` in bazel is equivalent to `if ... else ...`:
```bzl
select({ 
   "@config//:test_fp64_enabled": [ ],   # if
   "//conditions:default": [ 
      "ONEDAL_DISABLE_FP64_TESTS=1",     # else
   ],
})
```

Reply:
However, I agree this construction is confusing